test/modules/ftpserver_test.rb

```diff
@@ -21,39 +21,39 @@
 
 require_relative "../spec_helper.rb"
 
-stub_module("Users")
+describe "Yast::FtpServer" do
+  stub_module("Users")
 
-Yast.import "FtpServer"
+  Yast.import "FtpServer"
```

Comment:
this is bit uncommon to stub and import inside top level describe, what is rationale? ideally to comment if it have e.g. side-effects.

Reply:
I will try to move it to a `before` section. In the original test it was even worse because these lines were places before the `describe` section.

Reply:
in fact having it before describe is quite common as it has same strength as require call.